src/styles/init.js

```diff
@@ -5,10 +5,15 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import { StyleSheet } from 'react-native';
+import { Platform, StyleSheet } from 'react-native';
 import { PRIMARY_COLOR } from '../constants';
 
-const defaultBackgroundColor = '#fafafa';
+/**
+ * TODO: Obtain color from the OS scheme
+ * @see: https://reactnavigation.org/docs/5.x/themes/#using-the-operating-system-preferences
+ * @type {string}
+ */
+const defaultBackgroundColor = (Platform.OS === 'ios') ? '#fff' : '#fafafa';
```

Comment:
🚲 Shouldn't these colors `#fff` and `#fafafa` be in the `../constants` file as well?

Reply:
Good point. Refactored on [e4eb918](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/298/commits/e4eb9185b88bd150db7ccf2057b3763aa8622705).